front/src/modules/conflict/components/ConflictCardV2.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { BsLightningFill } from 'react-icons/bs';
+
+import type { ConflictV2 } from 'common/api/osrdEditoastApi';
+import type { TrainScheduleWithDetails } from 'modules/trainschedule/components/TimetableV2/types';
+import { stringToTime } from 'utils/date';
+
+export type ConflictTableProps = {
+  conflicts: ConflictV2[];
+  expanded: boolean;
+  trainSchedulesDetails: TrainScheduleWithDetails[];
+  toggleConflictsList: () => void;
+  onClick: (conflict: ConflictWithTrainNames) => void;
+};
+
+export type ConflictWithTrainNames = ConflictV2 & {
+  train_names: string[];
+};
+
+export default function ConflictCardV2({
+  conflict,
+  onClick,
+}: {
+  conflict: ConflictWithTrainNames;
+  onClick: (conflict: ConflictWithTrainNames) => void;
+}) {
+  const { t } = useTranslation(['operationalStudies/scenario']);
+
+  return (
+    <div className="conflict-card" onClick={() => onClick(conflict)} role="button" tabIndex={0}>
+      <BsLightningFill color="red" />
+      <div className="conflict-trains">
+        {conflict.train_names.map((train_name, idx) => (
+          <div className="card-text" key={`train-${idx}-${train_name}`}>
+            {train_name}
+          </div>
+        ))}
+      </div>
+      <div className="conflict-type">
+        <p>{t(`${conflict.conflict_type}`)}</p>
+      </div>
+      <div className="conflict-times">
+        <div className="start-time">{stringToTime(conflict.start_time)}</div>
+        <div className="end-time">{stringToTime(conflict.end_time)}</div>
```

Comment:
This won't work because you are resetting the date to 1/1/1900 and keep only the time.
As all start/end time are now dateTime and not only time, the date are given by the back as ISO8601 dates (the comment on the endpoint seems to be wrong). 
We already have a date util for that, it just only accepts number as parameters for now but you can add  `| string` to `msDate` type in parameters. You can also rename this function into : `formatIsoDate`.
```suggestion
        <div className="start-time">{msToIsoDate(conflict.start_time, true)}</div>
        <div className="end-time">{msToIsoDate(conflict.end_time, true)}</div>
```

Reply:
done